packages/cli/src/lifecycles/context.js

```diff
@@ -24,6 +27,8 @@ module.exports = initContexts = async({ config }) => {
       const userHasWorkspaceAppTemplate = await fs.exists(userAppTemplate);
       const userHasWorkspaceIndexTemplate = await fs.exists(path.join(userTemplatesDir, 'index.html'));
       const userHasWorkspaceNotFoundTemplate = await fs.exists(path.join(userTemplatesDir, '404.html'));
+      const userHasWorkspaceBabel = await fs.exists(path.join(config.workspace, babelConfig));
```

Comment:
I think default configuration / expectation for these kinds of files is that they are typically kept at the project root.

So instead of
```
<workspace>/
  babel.config.js
  pages/
  ...
```

It would make more sense to support
```
babel.config.js
<workspace>/
  pages/
  ...
```